cli_helpers/_compat.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""Python 2/3 compatibility support."""
```

Comment:
Sounds good, I dropped the underscore. Thanks for looking at this pull request. I'm going to go ahead and merge this in tomorrow as long as no other comments come up.